### ISSUE TITLE
Using LSTM for time series learning

### ISSUE THREAD

**Issue by StKlaas** [fchollet/keras#2781]
Hi, everybody, 
I want to train my model to learn time series. The input dimension is `(batch_size, 100, 23)`, 100 and 23 denote time steps and features at a time, respectively. The output dimension is `(batch_size, 100, 2)` as there are two label values to be learnt. Here is my code:

```
batch_size = 25
epochs = 100

 model = Sequential()
 model.add(LSTM(50,
                   batch_input_shape=(batch_size, 100, 23),
                   return_sequences=True,
                   stateful=True))
 model.add(LSTM(50,
                   batch_input_shape=(batch_size, 100, 23),
                   return_sequences=True,
                   stateful=True))
 model.add(TimeDistributed(Dense(2)))
 model.summary()
 
 model.compile(loss='mse',
                         optimizer='rmsprop')
 model.fit(X_train, y_train, batch_size=batch_size, nb_epoch=epochs,
               verbose=0) #,validation_data=(X_valid, y_valid))

```

The summary of the model is 
![image](https://cloud.githubusercontent.com/assets/17242434/15460856/32bf12ce-20e8-11e6-84d4-61412f29bab2.png)

However, it stops here and there is no more outputs. What's wrong with my model? 
Thank you in advance!

**Comment by geovedi** [comment]
is the problem still exists? i'm guessing your machine takes a longer time to compile the model. if that's the case, try open new terminal and monitor via `ps uxw` or `top -c`.

**Comment by StKlaas** [comment]
@geovedi the problem is that I set verbose=0. Now the model works.

**Comment by iwinterlovermy** [comment]
hi StKlaas, 
May I ask, the problem that you are solving, is it multiple steps ahead, or single step prediction? I currently have a multiple step ahead problem, I'm wondering if your code is applicable for multiple steps ahead.

Thanks

**Comment by StKlaas** [comment]
@iwinterlovermy It is a multiple steps ahead problem in my case. Hope it works with your problem.
